apps/frontend/src/components/Decorators/Draggable.tsx

```diff
@@ -8,16 +9,22 @@ interface DraggableProps {
 const Draggable: React.FunctionComponent<DraggableProps> = (props) => {
     const emitter = useMitt();
 
+    const dragImage = new Image();
+    dragImage.src = boxImage;
+
     const handleDragStart = (event: React.DragEvent) => {
         emitter.emit('dragStartEvent');
 
-        // Set cursor to "grab"
```

Comment:
Why did u remove the comment?

Reply:
What is done is obvious

Reply:
Not really.
I think is better to leave it

Reply:
Okay, I'll put it back